### PR TITLE
Add header

### LINKED ISSUE #17: [0.5pt] Add header - group task
Create page header according to the wireframe:

Screenshot 2021-06-27 at 21 42 24

### PR DESCRIPTION
**Pull request by Sboursen**
## Description

In this Branch

- [x] I added a basic header for the Home page. 

- [x] I added a basic style to the header

closes #17 

### other requirements

- [x] There are no linter errors.
- [x] Followed the **Gitflow**.

Thank you for taking the time to review this pull request.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,7 +8,23 @@
...
+    <nav class="flex justify-between items-center w-full">
+      <a class="block px-3" href="#">MyLgo</a>
+      <ul
+        class="flex justify-between items-center px-3 gap-6">
+        <li><a href="#">Spaceships <span>6</span></a></li>
+        <li><a href="#">Planets <span></span></a></li>
+        <li><a href="#">Races <span></span></a></li>
+      </ul>
+    </nav>
+  </header>
```
Comment (mwafrika) [inline src/index.html:25]: Good job so far @Sboursen, But there are some changes that you can make to the header, I think that it's better if you remove the padding to cover the whole space.

**Review by mwafrika — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (Sboursen) [reply src/index.html:25]: Hi @mwafrika 
Thank you for the feedback
I'm on it

**Comment by Sboursen** [comment]
Hi @mwafrika 
I made the requested changes:
image

Can you please review it again?

**Comment by mwafrika** [comment]
Hi @Sboursen,
Your project is complete, There is nothing other than saying it's time to merge it

**Review by mwafrika — APPROVED** [review]
@Sboursen,
Your project is complete, There is nothing other than saying it's time to merge it

**Pull request merged**